projects/aca-playwright-shared/src/api/sites-api.ts

```diff
@@ -24,20 +24,18 @@
 
 import { ApiClientFactory } from './api-client-factory';
 import { SiteBodyCreate, SiteEntry } from '@alfresco/js-api';
-import { users } from '../base-config/global-variables';
 import { logger } from '@alfresco/adf-cli/scripts/logger';
 import { SITE_VISIBILITY } from '../utils/configs';
```

Comment:
We already have SiteVisibility in the JS-API library, please use that one if possible

Reply:
done